docs/design/diagrams/subscribe_sequence.puml

```diff
@@ -9,7 +9,7 @@ entityAccessInfo:
 {
     name: "AmbientAirTemperature"
     id: "dtmi:sdv:HVAC:AmbientAirTemperature;1"
-    description: "..."
+    description: "The immediate surroundings air temperature (in Fahrenheit)."
```

Comment:
should be "The immediate surroundings' air temperature" (with an apostrophe after "surroundings"), though I'd suggest "The air temperature of the immediate surroundings" instead

Reply:
This was taken from the VSS spec.  Happy to fix it.

Reply:
Added the apostrophe.